.github/ISSUE_TEMPLATE/cherry-pick-request.yml

```diff
@@ -55,6 +55,19 @@ body:
         - LTS
     validations:
       required: true
+  - type: dropdown
+    id: formats
+    attributes:
+      label: Formats
+      description: The formats that this bug affects. See https://amp.dev/documentation/components/ for reference.
```

Comment:
No, because https://amp.dev/documentation/components/ has a sidebar that lists components available in each format, which I think is more helpful in this case than the "all about..." pages.

Reply:
Good point. (I was initially looking at the page on mobile, where the sidebar was hidden.)